airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```diff
@@ -252,14 +252,21 @@
           },
           {
             "title": "JSON Lines: Newline-delimited JSON",
-            "required": ["format_type"],
+            "required": ["format_type", "flattening"],
             "properties": {
               "format_type": {
                 "title": "Format Type",
                 "type": "string",
                 "enum": ["JSONL"],
                 "default": "JSONL"
               },
+              "flattening": {
+                "type": "string",
+                "title": "Normalization (Flattening)",
+                "description": "Whether the input json data should be normalized (flattened) in the output JSON Lines. Please refer to docs for details.",
```

Comment:
Nit: I know this is what the wording is for CSV, but I actually think the words "normalized" and "normalization" are being misused here. Let's call this what it is - flattening and remove "normalization" and "normalized" from both JSON and CSV options

Reply:
Agreed that this description should remove the use of `normalized` since its meaning is being misused, this should also apply to the description above for `CSV`

Another note is that there's no update to the docs for `JSONL` section so saying "Please refer to docs for details" may be a _little_ misleading unless they're keen on reading the `CSV` section

https://github.com/airbytehq/airbyte/blob/994ca25a0b449fca7d3973b6ba2334ede7bbd86f/docs/integrations/destinations/s3.md?plain=1#L229-L230